src/Chirp.Web/Pages/UserTimeline.cshtml

```diff
@@ -20,6 +20,13 @@
 {
     <div>
         <h2> @routeName's Timeline </h2>
+        <h3>Following:</h3>
+        <ul>
+            @foreach (Author followingAuthor in Model.user.Following)
+            {
+                <li>@followingAuthor.UserName</li>
```

Comment:
Why are we displaying every username here?

Reply:
Nice spot! That is redundant debugging code. Will be removed